src/containers/my-courses/course-toolbar/CourseToolbar.style.ts

```diff
@@ -26,11 +26,16 @@ export const styles = {
     position: 'relative',
     '& .MuiFormHelperText-root': {
       ...commonStyle
+    },
+    '@media (max-width: 1220px)': {
```

Comment:
try to use xs, sm, md, lg for adaptivity

Reply:
> try to use xs, sm, md, lg for adaptivity

rewrited with mui adaptivity